### ISSUE TITLE
[Bug] Error when selecting RemoteTech mod that causes all other mod selections to fail also

### ISSUE THREAD

**Issue by wylker** [KSP-CKAN/CKAN#2861]
Background
----------
* Operating System: win 10
* CKAN Version: 1.26.4
* KSP Version: 1.7.3.2594

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
no

Problem
-------
When selecting RemoteTech for installation (simply checking the checkbox), the following error occurs:

**CKAN error code (if applicable):**

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: rowIndex
   at System.Windows.Forms.DataGridView.InvalidateRow(Int32 rowIndex)
   at CKAN.Main.ConflictsUpdated(Dictionary`2 prevConflicts)
   at CKAN.Main.set_Conflicts(Dictionary`2 value)
   at CKAN.Main.<UpdateChangeSetAndConflicts>d__63.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CKAN.Main.<ModList_CellValueChanged>d__284.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)

**Comment by HebaruSan** [comment]
Duplicate of #2852

**Comment by HebaruSan** [comment]
Thanks for submitting this, we have looked into it and have a fix in #2854.
